src/main/java/ja/crawler/Entities/BookInfo.java

```diff
@@ -0,0 +1,25 @@
+package ja.crawler.Entities;
+
+public class BookInfo {
+
+
+    String title;
```

Comment:
It has a getter and package access, please choose one option. Also lines 4-9 too many empty lines.